[Guide updates] Renew DigiCert certificates on macOS (#28077)
fleetdm/fleet: articles/connect-end-user-to-wifi-with-certificate.md

When Fleet delivers the profile to your hosts, Fleet will replace the variables. If something goes wrong, errors will appear on each host's **Host details > OS settings**.

Fleet will automatically resend the profile to renew the certificate 30 days before expiration.

If a variable used in step 3 above has changed, Fleet will resend the profile. If you used a variable like `$FLEET_VAR_HOST_END_USER_IDP_USERNAME` for CN and seat ID and it changed, Fleet will get a new certificate and create a new seat in DigiCert, which will take 1 license. If you want to revoke a license in DigiCert, head to [**Trust Lifcycle Manager > Account > Seats**](https://demo.one.digicert.com/mpki/account/seats) and remove the seat.

> If the device gets deleted from Fleet, it will still take seat in DigiCert. Follow the instructions above to revoke a license.

Author: ```suggestion
If the device gets deleted from Fleet, it will still take seat in DigiCert. You can tell the team that owns DigiCert to follow the instructions above to revoke a license.
```

When Fleet delivers the profile to your hosts, Fleet will replace the variables. If something goes wrong, errors will appear on each host's **Host details > OS settings**.

Fleet will automatically resend the profile to renew the certificate 30 days before expiration.

> If end user is on vacation (offline more than 30 days) and certificate expires, they should connect to a different network so new certificate is delivered and then they can connect back to a corporate Wi-Fi and VPN.

Reviewer: ```suggestion
If an end user is on vacation (offline more than 30 days), their certificate might expire and they'll lose access to Wi-Fi or VPN. To get them reconnected, ask your end users to momentarily connect to a different network so that Fleet can deliver a new certificate.
```

Language tweaks here. What do you think?

Author: LGTM!

> DigiCert device type seats can have multiple certificates assigned, and the user will pay for only one license. Only duplicate certificates (those with the same CN and seat ID) can be assigned to a one device type seat.

Reviewer: ```suggestion
Each DigiCert device type seat (license) can have multiple certificates only if they have the same CN and seat ID. If a new certificate has a different CN, a new DigiCert license is required.
```

Tried to make this understandable by someone who knows nothing about DigiCert. Accurate?

> If value of any variable used in step 3 above has changed, Fleet will resend the profile. If you used a variable like `$FLEET_VAR_HOST_END_USER_IDP_USERNAME` for CN and seat ID and it changed, Fleet will get a new certificate and create a new seat in DigiCert, which will take one license. If you want to revoke a license in DigiCert, head to [**Trust Lifcycle Manager > Account > Seats**](https://demo.one.digicert.com/mpki/account/seats) and remove the seat.

Reviewer: ```suggestion
If the value for any variable used in step 3 above has changes, Fleet will resend the profile. This means, if you use a variable like `$FLEET_VAR_HOST_END_USER_IDP_USERNAME` for CN or seat ID, and the variable's value changes, Fleet will get a new certificate and create a new seat in DigiCert. This will add a new DigiCert license. If you want to revoke a license in DigiCert, head to [**Trust Lifcycle Manager > Account > Seats**](https://demo.one.digicert.com/mpki/account/seats) and remove the seat.
```

> If the device gets deleted from Fleet, it will still take seat in DigiCert. You can tell the team that owns DigiCert to follow the instructions above to revoke a license.

Reviewer: ```suggestion
DigiCert seats aren't automatically revoked when hosts are deleted in Fleet. To revoke a license, ask the team that owns DigiCert to follow the instructions above.
```

### Step 2: Add SCEP configuration profile to Fleet

1. Create a [configuration profile](https://fleetdm.com/guides/custom-os-settings) with the SCEP payload. In the profile, for `Challenge`, use`$FLEET_VAR_NDES_SCEP_CHALLENGE`. For `URL`, use `$FLEET_VAR_NDES_SCEP_PROXY_URL`, and make sure to add `$FLEET_VAR_SCEP_RENEWAL_ID` to `CN`.

> > If you previously added a profile without `$FLEET_VAR_SCEP_RENEWAL_ID` it will work, but automati renewal won't be enabled.

Reviewer: ```suggestion
```

I think we can get away without documenting this. Why? DigiCert is experimental. We want all IT admins to use automatic renewal.

### Step 2: Add SCEP configuration profile to Fleet

1. Create a [configuration profile](https://fleetdm.com/guides/custom-os-settings) with the SCEP payload. In the profile, for `Challenge`, use`$FLEET_VAR_CUSTOM_SCEP_CHALLENGE_<CA_NAME>`. For, `URL`, use `$FLEET_VAR_CUSTOM_SCEP_PROXY_URL_<CA_NAME>`, and make sure to add `$FLEET_VAR_SCEP_RENEWAL_ID` to `CN`.

> > If you previously added a profile without `$FLEET_VAR_SCEP_RENEWAL_ID` it will work, but automati renewal won't be enabled.

Reviewer: ```suggestion
```
